storage/rocksdb/rdb_datadic.cc

```diff
@@ -964,6 +964,15 @@ bool Rdb_key_def::covers_lookup(TABLE *const table,
   return bitmap_is_subset(lookup_bitmap, &covered_bitmap);
 }
 
+/* Indicates that all key parts can be unpacked to cover a secondary lookup */
```

Comment:
It looks like it is necessary to call can_cover_lookup() before each covers_lookup() call. Fold the former into the latter, by inlining or by having a private method?